spec/controllers/campaign_pages_controller_spec.rb

```diff
@@ -1,28 +1,77 @@
 require 'rails_helper'
-require 'capybara/poltergeist'
-require 'helper_functions'
-
-RSpec.describe CampaignPagesController, type: :feature do
-  scenario 'Redirect after a user signs the petition' do
-    cam_page = create_petition_page
-    visit "/campaign_pages/#{cam_page.id}"
-    expect(find('#redirect-location')).to have_text 'www.google.com'
-  end
 
-  scenario 'Index displays the list of active campaign pages' do
-    cam_page = create_petition_page
-    log_in
-    visit '/campaign_pages'
-    expect(page.body).to have_text cam_page.title
-  end
+RSpec.describe CampaignPagesController, type: :controller do
```

Comment:
You can drop `Rspec.describe` and `type: :controller`.

`describe CampaignPagesController do` works just fine, so long as the file is within `spec/controllers`